src/components/Chart/DigitalChannels.jsx

```diff
@@ -6,13 +6,22 @@
 
 import React from 'react';
 import { Line } from 'react-chartjs-2';
+import {
+    Chart as ChartJS,
+    LinearScale,
+    LineElement,
+    PointElement,
+    Title,
+} from 'chart.js';
 import { colors } from 'pc-nrfconnect-shared';
 import { arrayOf, bool, exact, number, shape } from 'prop-types';
 
 import crossHairPlugin from './plugins/chart.crossHair';
 
 import chartCss from './chart.icss.scss';
 
+ChartJS.register(LineElement, PointElement, LinearScale, Title);
```

Comment:
Oh, I can see now, that it doesn't collide with the other file registering the same components :)